apps/api/src/app/widgets/usecases/get-notifications-feed/get-notifications-feed.usecase.ts

```diff
@@ -103,21 +103,17 @@ export class GetNotificationsFeed {
     const hasMore = feed.length < totalCount;
     totalCount = Math.min(totalCount, command.limit);
 
+    const data = feed.map((el) => ({ ...el, content: el.content as string }));
```

Comment:
I didn't want to break the logic with different implementation that's why forced the type of the content to be string as for the `in-app` it's always string